.github/workflows/docker_utils.yml

```diff
@@ -19,10 +18,6 @@ on:
         required: false
         type: string
         default: stable
-      features:
```

Comment:
if you remove this you need to remove/update the related steps for the tag generation

Reply:
You mean this can be removed?

https://github.com/topos-protocol/topos/pull/352/files#diff-d375c297193279683833c8534db2366059cefe310fb8a33c6bba4b765492c145L76